system_test/aest_sync_SUITE.erl

```diff
@@ -571,6 +678,17 @@ quick_start_stop(Cfg) ->
 
 %% helper functions
 
+node_mined_retries(Nodes) ->
+    Metric = "ae.epoch.aecore.mining.retries.value",
```

Comment:
Not sure about this metric:
https://github.com/aeternity/epoch/blob/2edd9fbe4e95bc010d57e043954d930ac7beba7f/apps/aecore/src/aec_conductor.erl#L693
https://github.com/aeternity/epoch/blob/2edd9fbe4e95bc010d57e043954d930ac7beba7f/apps/aecore/src/aec_conductor.erl#L698

Reply:
I see - nonce attempted without success. Thanks @sylane @dincho